### PR TITLE
OP-445 Disable telemetry by default on installs

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
Signed-off-by: Piyush Nimbalkar <pnimbalkar@purestorage.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/storage/portworx/portworx_test.go [hunk 1]
```diff
@@ -6621,16 +6619,22 @@ func TestStorageClusterDefaultsForTelemetry(t *testing.T) {
...
+	require.False(t, cluster.Spec.Monitoring.Telemetry.Enabled)
 	require.Empty(t, cluster.Status.DesiredImages.Telemetry)
 
-	// Enable telemetry by default for 2.8.0
+	// Allow telemetry to be enabled for 2.8.0
 	cluster.Spec.Image = "px/image:2.8.0"
+	cluster.Spec.Monitoring.Telemetry.Enabled = true
 	driver.SetDefaultsOnStorageCluster(cluster)
 	require.NotEmpty(t, cluster.Spec.Monitoring) // telemetry is under monitoring
 	require.NotEmpty(t, cluster.Spec.Monitoring.Telemetry)
```
Comment (dsupure) [inline drivers/storage/portworx/portworx_test.go:6640]: should we change this?
```suggestion
      require.True(t, cluster.Spec.Monitoring.Telemetry.Enabled)
```

Comment (piyush-nimbalkar) [reply drivers/storage/portworx/portworx_test.go:6640]: We are doing that check on the next line.

Comment (dsupure) [reply drivers/storage/portworx/portworx_test.go:6640]: Ah right. Didn't see it in the review mode.

**Review by dsupure — APPROVED** [review]
LGTM!

**Pull request merged**